lib/big_query/base/timestamp.ex

```diff
@@ -29,11 +29,13 @@ defmodule BigQuery.Base.Timestamp do
   end
 
   def timestamp_params(interval) do
+    lower = interval.rollup.floor(interval.from)
```

Comment:
In theory, the Interval Plug should round the from/to query params.  But it doesn't hurt to make sure here.  (Don't want to query/cache partial days of data, or something).